LLMonFHIR/Resources/en.lproj/Localizable.strings

```diff
@@ -105,6 +108,7 @@ The following JSON representation defines the FHIR resource that you should inte
 Provide a short one-sentence summary of the resource in less than 20 words.
 Do NOT respond with more content than the single line containing the summary.
 Directly provide the content without any additional structure.
+Chat with the user in
```

Comment:
You can remove the `Chat with the user in` line now that we append it outside of the prompt. Same for all other languages as well.